templates/config.yml

```
@@ -0,0 +1,51 @@
+conda/governance:
+  # [required] community files
+  - src: CODE_OF_CONDUCT_SHORT.md
+    dst: CODE_OF_CONDUCT.md
```

```suggestion
  - CODE_OF_CONDUCT.md
```

@kenodegard Is that the right way to write it then assuming we don't have a dedicated short version of the Coc?